### PR TITLE
Renaming the .vsix.

### PR DESCRIPTION
**Pull request by raquelsa**

### CODE REVIEW

File: src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj [hunk 1]
```diff
@@ -25,6 +25,7 @@
     <AssemblyName>Microsoft.Web.LibraryManager.Vsix</AssemblyName>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <GeneratePkgDefFile>true</GeneratePkgDefFile>
+    <TargetVsixContainerName>Microsoft.Web.LibraryManager.Vsix</TargetVsixContainerName>
```
Comment (jimmylewis) [inline src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj]: nit: the file extension should be lower case for consistency

Comment (raquelsa) [reply src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj]: Missed that...

**Comment by joeloff** [comment]
I can't leave comments in the files, so here goes:

<ExtensionInstallationFolder>Microsoft\Web Tools Azure\WebJobs</ExtensionInstallationFolder> This should be a folder for the Library installer. This folder you have is used by WTA extensions, it was just an example.

Also, just be aware that once IsProductComponent is set, people can't build and install the VSIX locally, because it will expect to have a matching component wrapper in the VS setup manifest - this won't happen until we do the actually insertion and ship VS

**Comment by joeloff** [comment]
For the versioning <Identity Id="Microsoft.Web.LibraryManager" Version="15.0.0" Language="en-US" Publisher="Microsoft" /> Do you want to manually change it or do it automatically?

You can inject properties, assuming you have a way to generate a build number on the fly. I can share some examples.

File: appveyor.yml [hunk 2]
```diff
@@ -18,4 +18,7 @@ artifacts:
     name: NuGet package
 
   - path: src\**\*.vsix
-    name: Visual Studio Extension
\ No newline at end of file
+    name: Visual Studio Extension
+
+    - path: src\**\Microsoft.Web.LibraryManager.json
+    name: Visual Studio Extension manifest
```
Comment (jimmylewis) [inline appveyor.yml]: nit: always put a newline at the end of the file.

**Pull request merged**